pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -103952,8 +103952,8 @@ self: {
      }:
      mkDerivation {
        pname = "hocker";
-       version = "1.0.0";
-       sha256 = "16indvxpf2zzdkb7hp09zfnn1zkjwc1pcg2560x2vj7x4akh25mv";
+      version = "1.0.2";
```

Comment:
The new version fails its test suite on NixOS: https://nix-cache.s3.amazonaws.com/log/fb2a5pp8l6k1ys5gjvj1djh9r8ba27bk-hocker-1.0.2.drv

Reply:
Thank you for pointing that out, I had a late night oopsie and forgot to include an override that fixes this issue. I also need to take the temporary override I put in place in the test's `machine.nix`.

I also see the `hackage-packages.nix` has been updated so I'll drop this hunk.

Reply:
It's failing its test suite due to a missing data file :-/ I thought I'd fixed that but apparently not. Since we need to override the derivation to wrap the programs with a PATH to nix I added a `dontCheck` for now; when I fix this fully I'll open another PR to remove the `dontCheck`.